awesome/src/widgets/systray.lua

```diff
@@ -8,16 +8,19 @@ local color = require("src.theme.colors")
 local dpi = require("beautiful").xresources.apply_dpi
 local gears = require("gears")
 local wibox = require("wibox")
-require("src.core.signals")
 
+local capi = {
+    awesome = awesome,
+}
 
-return function(s)
+require("src.core.signals")
 
+return function(s)
     local systray = wibox.widget {
         {
             {
                 wibox.widget.systray(),
-                margins = dpi(6),
+                margins = dpi(0),
```

Comment:
The margin isnt needed anymore, please remove the margin container.

Reply:
:thumbsup:

Reply:
I pushed the fixes but I am not really sure that I did this one right, since you specifically mentioned the container